StylableWinFormsControls/StylableWinFormsControls/Controls/StylableCheckBox.cs

```diff
@@ -143,70 +141,25 @@ private Rectangle getTextRectangle(Size glyphSize)
         return _textRectangleValue;
     }
 
-    private static ButtonState convertToButtonState(CheckBoxState state)
+    private ButtonState convertToButtonState()

```

Comment:
I'd recommend a slightly different naming for `convertToButtonState` and `convertToCheckBoxState` (e.g. `getButtonState` or similar) - if someone reads the signature, they might ask themselves how you would convert the whole checkbox to a state :smile:  
Otherwise, this looks good in code and tests.